Workbooks/CosmosDb/Resource Insights/Resource Insights.workbook

```diff
@@ -67,16 +67,93 @@
               "durationMs": 86400000
             },
             "value": {
-              "durationMs": 86400000
+              "durationMs": 14400000
             }
           },
+          {
+            "id": "d39fabbe-e275-42c7-bf2c-dda5a139eb82",
+            "version": "KqlParameterItem/1.0",
+            "name": "EnabledApiTypes",
+            "type": 1,
+            "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"{Resources}\",\"urlParams\":[{\"key\":\"api-version\",\"value\":\"2021-01-15\"}],\"batchDisabled\":false,\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"tablePath\":\"properties.EnabledApiTypes\",\"columns\":[]}}]}",
+            "isHiddenWhenLocked": true,
+            "queryType": 12
+          },
+          {
+            "id": "3bd5e0c6-988c-4e44-81d1-ed78c142f18c",
+            "version": "KqlParameterItem/1.0",
+            "name": "ApiDatabases",
+            "type": 1,
+            "value": "sqlDatabases",
+            "isHiddenWhenLocked": true,
+            "criteriaData": [
+              {
+                "criteriaContext": {
+                  "leftOperand": "EnabledApiTypes",
+                  "operator": "==",
+                  "rightValType": "static",
+                  "rightVal": "Sql",
+                  "resultValType": "static",
+                  "resultVal": "sqlDatabases"
+                }
+              },
+              {
+                "criteriaContext": {
+                  "leftOperand": "EnabledApiTypes",
+                  "operator": "contains",
```

Comment:
should the operators here just be ==?

Reply:
The reason for the contains is that these API types (besides SQL) have both SQL and their respective API Type. For example, for Gremlin, you will see ("Sql","Gremlin").